zgw/notificaties-api/src/main/kotlin/com/ritense/notificatiesapi/autoconfigure/NotificatiesApiAutoConfiguration.kt

```diff
@@ -56,4 +61,37 @@ class NotificatiesApiAutoConfiguration {
             abonnementLinkRepository
         )
     }
+
+    @Bean
+    @ConditionalOnMissingBean(WebClient::class)
+    fun notificatiesApiWebClient(): WebClient {
```

Comment:
We stopped creating webclient beans since yesterday. See: [this PR](https://github.com/valtimo-platform/valtimo-backend-libraries/pull/520/files#diff-3f9d5687e61501ceebbdfc620222693f7ef0b05334d458ca8ddf6a81adeaaf74L50-L71)

Reply:
That was already there..what should be done then?

Reply:
This webclient bean should be removed